### DISCUSSION TITLE
本地windows无法登录

### DISCUSSION THREAD

**Discussion by victoryzzg** [illacloud/illa-builder#604] [Q&A]
环境信息：
1. WIN10
2. 安装了Docker
3. 安装了illa
现状：
1. 可以通过localhost:10000访问
 问题：
1. 无法登录
2. 无法注册，发送验证码失败

**Comment by karminski** [comment]
@victoryzzg 你好，请问是如何安装的？如果直接拉镜像是没办法运行起来的，需要配置环境变量。
可以使用我们的部署工具：
- https://github.com/illacloud/illa
或者部署脚本：
- https://github.com/illacloud/deploy-illa-manually/tree/main/docker
